packages/0x.js/test/utils/fill_scenarios.ts

```diff
@@ -21,6 +26,26 @@ export class FillScenarios {
         this.zrxTokenAddress = zrxTokenAddress;
         this.exchangeContractAddress = exchangeContractAddress;
     }
+    public async initTokenBalancesAsync() {
+        const web3Wrapper = (this.zeroEx as any)._web3Wrapper as Web3Wrapper;
+        for (const token of this.tokens) {
+            if (token.symbol === 'WETH') {
+                // _.noop
+            } else if (token.symbol !== 'ZRX') {
+                const contractInstance = web3Wrapper.getContractInstance(
+                    artifacts.DummyTokenArtifact.abi, token.address,
+                );
+                const dummyToken = new DummyTokenContract(contractInstance, {});
+                const tokenSupply = ZeroEx.toBaseUnitAmount(INITIAL_COINBASE_TOKEN_SUPPLY_IN_UNITS, token.decimals);
+                const txHash = await dummyToken.setBalance.sendTransactionAsync(this.coinbase, tokenSupply, {
+                    from: this.coinbase,
+                });
+                await this.zeroEx.awaitTransactionMinedAsync(txHash);
+            } else {
+                // no-op
+            }
```

Comment:
This can be written as a single conditional:

```
if (token.symbol !== 'WETH' && token.symbol !== 'ZRX') {
    const contractInstance = web3Wrapper.getContractInstance(
        artifacts.DummyTokenArtifact.abi, token.address,
    );
    const dummyToken = new DummyTokenContract(contractInstance, {});
    const tokenSupply = ZeroEx.toBaseUnitAmount(INITIAL_COINBASE_TOKEN_SUPPLY_IN_UNITS, 
    token.decimals);
    const txHash = await dummyToken.setBalance.sendTransactionAsync(this.coinbase, tokenSupply, {
        from: this.coinbase,
     });
     await this.zeroEx.awaitTransactionMinedAsync(txHash);
}

```